src/main/java/com/mide/gangsaeng/bannedword/BannedWordServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package com.mide.gangsaeng.bannedword;
+
+import java.util.List;
+import java.util.concurrent.ConcurrentHashMap;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class BannedWordServiceImpl implements BannedWordService {
+
+    private static ConcurrentHashMap<String,BannedWord> cache;
+
+    public BannedWordServiceImpl() {
+        cache = new ConcurrentHashMap<>();
+    }
+
+    @Override
+    public void add(String word) {
+        if (isBanned(word)) {
+            throw new IllegalArgumentException("Already added.");
```

Comment:
Can we give appropriate HTTP status code to client when the word is already registered?

Reply:
It's just a business logic error, so I think 200 is appropriate with a notification message. 
In this case, I chose to return a simple `String` message.

Reply:
I plan to improve it later. (Using custom exception.)